actions/templates/actions/version.html

```diff
@@ -34,6 +34,18 @@ <h1 class="text-2xl leading-6 font-mono font-bold text-gray-900">
             {% endfor %}
           </dd>
         </div>
+        {% if action.topics %}
+        <div class="sm:col-span-2">
+          <dt class="text-sm font-semibold text-gray-600">
+            Topics
+          </dt>
+          <dd class="mt-1 text-sm text-gray-900">
+            {% for topic in action.topics %}
+                {{ topic }}{% if not forloop.last %}, {% endif %}
+            {% endfor %}
+          </dd>
+        </div>
+        {% endif %}
```

Comment:
Why the logic? It's useful to see when an action doesn't have any topics, and not have to guess at the underlying logic.